pages/configuration/multinode_setup.rst

```diff
@@ -99,9 +97,9 @@ Each component in this multi-node setup can be scaled on the individual needs.
 
 Depending on the amount of messages ingested and how long messages should be available for direct search, the Elasticsearch cluster will need most of the resources on your setup.
 
-Keep an eye on your Elasticsearch cluster with plugins like `Elastic HQ <http://www.elastichq.org>`__ or `Kopf <https://github.com/lmenezes/elasticsearch-kopf>`__. Those will help you to understand the Elasticsearch cluster health and behavior.
+Keep an eye on the Metrics of each part of the cluster. One option is to use `telegraf <https://github.com/influxdata/telegraf>`__ to fetch importand metrics and store them in your favorite metric system (e. g. Graphite, Prometheus or Influx).
```

Comment:
I would keep the links to Kopf and Elastic HQ, since they're widely used plugin and not everyone is using Telegraf or the TICK stack.